content/en/project/status/_index.md

```diff
@@ -7,18 +7,26 @@ weight: 50
 ## Annual Reports
 
 The Medley team has produced annual reports documenting their major achievements.
```

Comment:
If you want this on a separate line from the **Sneak Peak:** sentence add a couple spaces to the end.  If you would also like a paragraph break and a new line with 2 spaces in it.